content/en/docs/tasks/administer-cluster/dns-debugging-resolution.md

```diff
@@ -341,10 +341,10 @@ be lost. To work around this limit, the node can run `dnsmasq`, which will
 provide more `nameserver` entries. You can also use kubelet's `--resolv-conf`
 flag.
 
-If you are using Alpine version 3.3 or earlier as your base image, DNS may not
-work properly due to a known issue with Alpine.
-Kubernetes [issue 30215](https://github.com/kubernetes/kubernetes/issues/30215)
-details more information on this.
+If you are using Alpine version 3.17 or earlier as your base image, DNS may not
+work properly due to a design issue with Alpine. 
```

Comment:
nit: I prefer `known` instead of `design`

nit: I'm finding it hard to fully understand the sentence that begins with `Until`.
Is this correct:
Since version 1.24 of the `musl` library did not include TCP fallback to the DNS stub resolver, any DNS requests larger than 512 bytes may fail.

nit: Could you remove the word `Please` from the last sentence.

Reply:
Hi Karen,

Unfortunately we can't because it was never an issue was a choice aka design as the musl libc creator attest:

![image](https://github.com/kubernetes/website/assets/47302125/2f4862f9-031c-432c-b74d-4958721a96e4)

With all due respect one of the biggest issues on Kubernetes documentation is being based on preferences instead of technical facts and that reflects its quality or in this case the lack of it.

Thanks.

Reply:
Thank you @everonline  for bringing this up